modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -65,7 +65,7 @@ static int toNCDHW(int idx)
 }
 
 // This values are used to indicate layer output's data layout where it's possible.
-enum DataLayout
+enum DataLayoutTF
```

Comment:
The `enum DataLayout` in `dnn.hpp` conflict with the same enum in `tf_importer.cpp` and `tflite_importer.cpp`
Change the name from `DataLayout` to `DataLayoutTF` to handle this issue.

Reply:
I'm against of introducing another data type. Let's reuse DataLayout enumeration